src/sentry/api/bases/organization_events.py

```diff
@@ -294,22 +294,33 @@ def get_event_stats_data(
                 # level thresholds in the request and use the legacy apdex calculation.
                 # TODO(snql): Alias the project_threshold_config column so it doesn't
                 # have to be in the SELECT statement and group by to be able to use new apdex.
```

Comment:
Does this comment need to be updated to reflect `user_misery()` and `count_miserable(user)`?